deployment/v3/docs/change-domain-name-in-a-running-env.md

```diff
@@ -0,0 +1,96 @@
+# Change domain name in a running env ( v2 )
```

Comment:
# Change domain name in a running environment (V2)

Check if it needs to be V2 or v2